### PR TITLE
initial commit to add shopify / e-commerce support

### PR DESCRIPTION
**Pull request by lbirdeau**
# What
Adds support for Shopify and similar e-commerce retailers.

# Why 
Shopify and similar e-commerce sites do not allow changes to the underlying HTML. This update changes how DOM elements and updates are identified, making it possible to enrich elements without editing the source HTML.

# Bug Fixes
- Input elements using the inline-block display style now retain this style once converted to a combo box. 
- 401 Invalid Key errors returned from Lob's US verification endpoint now trigger the default form submission.

### CODE REVIEW

File: src/lob-address-elements.js [hunk 1]
```diff
@@ -1,527 +1,724 @@
...
+            primary: parts.primary_line,
+            secondary: parts.secondary_line,
+            city: data.components && data.components.city || '',
+            state: data.components && data.components.state || '',
+            zip: plus4(data.components)
+          }
+          for (var p in address) {
+            if (address.hasOwnProperty(p)) {
+              if (address[p].toUpperCase() != config.elements[p].val().toUpperCase() &&
+                !(p === 'zip' && config.elements[p].val().length === 5 && address[p].indexOf(config.elements[p].val()) == 0)) {
```
Comment (leifg) [inline src/lob-address-elements.js:456]: `config.elements[p].val()) == 0`

Isn't that supposed to be triple equal?

File: src/lob-address-elements.js [hunk 2]
```diff
@@ -1,527 +1,724 @@
...
-                            !(p === 'zip' && config.elements[p].val().length === 5 && address[p].indexOf(config.elements[p].val()) == 0)) {
-                            didFix = true;
-                        }
-                        config.elements[p].val(address[p]);
-                    }
-                }
-                return didFix;
-            }
+        /**
+         * Show form- and field-level error messages as cofigured. Verification did NOT succeed.
```
Comment (leifg) [inline src/lob-address-elements.js:519]: Typo `cofigured` (missing n)

File: README.md [hunk 3]
```diff
@@ -12,368 +13,405 @@ The Lob Address Elements library simplifies client side integration of Lob's *US
...
 
+## Releases
+
+[Minified builds](https://github.com/lob/lob-address-elements/tree/master/lib) map to the releases listed below.
+
+### 1.1.0 (CURRENT / LATEST)
+| Release Notes |
+| :---          |
+| Target HTML elements are enriched in real time as soon as they appear in the DOM. |
+| Target HTML elements are indentifiable using a flexible addressing scheme. This includes element IDs, element names, and in-line data attributes. |
```
Comment (leifg) [inline README.md:405]: typo `indentifiable` (n too much)

File: src/lob-address-elements.js [hunk 4]
```diff
@@ -1,527 +1,724 @@
...
+          city_state_zip: function (msg) {
+            config.elements.cityMsg.text(msg).show('slow');
+            config.elements.stateMsg.text(msg).show('slow');
+            config.elements.zipMsg.text(msg).show('slow');
+          },
+          zip: function (msg) {
+            config.elements.zipMsg.text(msg).show('slow');
+          },
+          deliverable_missing_unit: function (msg) {
+            (config.denormalize === false &&
```
Comment (leifg) [inline src/lob-address-elements.js:544]: Is there a reason we're not using `!config.denormalize`

File: src/lob-address-elements.js (hunk 1 above)
Comment (lbirdeau) [reply src/lob-address-elements.js]: good catch

File: src/lob-address-elements.js (hunk 2 above)
Comment (lbirdeau) [reply src/lob-address-elements.js]: fixed

File: README.md (hunk 3 above)
Comment (lbirdeau) [reply README.md]: fixed

File: src/lob-address-elements.js (hunk 4 above)
Comment (lbirdeau) [reply src/lob-address-elements.js]: good catch. fixed in 3 other places as well.

**Review by leifg — APPROVED** [review]

**Pull request merged**